task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "appstudio, hacbs"
+  name: fbc-validation
+spec:
+  params:
+    - name: IMAGE_URL
+      description: the fully qualified image name
+  results:
+    - name: HACBS_TEST_OUTPUT
+  steps:
+    - name: check-labels-and-binaries
+      image: quay.io/redhat-appstudio/hacbs-test:latest
+      workingDir: /work
+      securityContext:
+        runAsUser: 1000
+      env:
+        - name: HOME
+          value: /work
+      resources:
+        limits:
+          memory: 4Gi
+          cpu: 2
+        requests:
+          memory: 512Mi
+          cpu: 10m
+      script: |
+        ### Try to extract binaries with configs > check binaries functionality > check opm validate ###
+
+        mkdir confdir
+        if ! oc image extract $(params.IMAGE_URL) --file /bin/opm --file /bin/grpc_health_probe --path configs/*:confdir/ ; then
```

Comment:
About `-path configs/*:confdir/` - I think we still need to consult the `operators.operatorframework.io.index.configs.v1` image label to get the location of the configs directory.
Since the `sanity-inspect-image` task gets the image info, we can consult the image inspect json to get the label value like it's done in the sanity-label-check task [here](https://github.com/redhat-appstudio/build-definitions/blob/main/task/sanity-label-check/0.1/sanity-label-check.yaml#L35).

Reply:
made the changes so it uses `../sanity-inspect-image/image_inspect.json` and parses the configs folder from here